src/main/eo/org/eolang/txt/text.eo

```diff
@@ -69,3 +69,11 @@
 
   [i] > at
     ^.s.slice i 1 > @
+
+  # Returns
+  [] > as-int
+    at. > @
+      QQ.txt.sscanf
```

Comment:
@levBagryansky I think it would be better to explicitly declare that we use `sscanf` as alias

Reply:
@Graur Ok